src/utils/fetchChefPairInfo.ts

```diff
@@ -1,18 +1,34 @@
 import { NetworkId } from '../constants'
-import { pairQuery } from '../graphql/query'
-import { voltageClient } from '../graphql'
+import { pairQuery, stablePoolQuery } from '../graphql/query'
+import { stableswapClient, voltageClient } from '../graphql'
+import { isStableswap, weiToNumber } from '.'
+import fetchTokenPrice from './fetchTokenPrice'
 
 async function fetchPairInfoVoltage (address: string) {
-  const result = await voltageClient.query({
-    query: pairQuery(address)
-  })
-  return {
-    reserveUSD: result?.data?.pair?.reserveUSD,
-    totalSupply: result?.data?.pair?.totalSupply,
-    token0: result?.data?.pair?.token0,
-    token1: result?.data?.pair?.token1,
-    totalReserve0: result?.data?.pair?.reserve0,
-    totalReserve1: result?.data?.pair?.reserve1
+  let result
+  if (isStableswap(address)) {
+    result = await stableswapClient.query({
+      query: stablePoolQuery(address)
+    })
+    const tokens = result?.data?.swaps?.tokens?.map((token) => token?.id)
+    const reserves = result?.data?.swaps?.balances
+    const pooledTokensUsd = result?.data?.swaps?.balanes?.map(async (balance, i) => (await fetchTokenPrice(tokens[i], NetworkId.FUSE)) * balance)
+    return {
+      reserveUSD: pooledTokensUsd?.reduce((mem, balanceUsd, i) => mem + weiToNumber(balanceUsd, result?.data?.swaps?.tokens?.[i]?.decimals)),
```

Comment:
@mul53 use virtualPrice instead?

Reply:
we can use `lpTokenSupply * 1` for the tvl